backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/entities/Shop.java

```diff
@@ -0,0 +1,86 @@
+package com.google.gpay.virtualqueue.backendservice.entities;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+public class Shop {
+    private String shopkeeperUid;
+    private String shopOwnerName;
+    private String shopName;
+    private String address;
+    private String phnNo;
+    private Integer lastTokenAlloted;
+    private String shopType;
+    private Integer shopId;
+    private List<UUID> tokens = new ArrayList<>();
+    public Shop(){
+    }
+    public Shop(String shopkeeperUid, String shopOwnerName, String shopName, String address, String phnNo, String shopType){
+        super();
+        this.shopkeeperUid=shopkeeperUid;
```

Comment:
Nit. Add ' ' before and after '=' operators. 
This applies everywhere for all operators.